spec/interactions/users/authenticate_user_spec.rb

```diff
@@ -2,21 +2,26 @@
 
 describe AuthenticateUser, type: :interactions do
 
-  let!(:user) { FactoryGirl.create(:user) }
 
-  subject { UserService.authenticate!(user.email) }
+  context 'when user exists' do
+    let(:user) { create(:user) }
 
-  it 'finds for a user' do
-    expect(User).to receive(:find_by).with({email: user.email})
-    expect(subject).to eq(user)
-  end
+    subject { described_class.run(user.attributes).result }
 
-  context 'when user exists' do
-    it 'returns user'
+    it 'finds for a user' do
+      is_expected.to eq(user)
+    end
   end
 
   context 'when user does not exists' do
-    it 'creates a new user'
+    let(:attrs) { { 'first_name' => 'John', 'last_name' => 'Doe', 'email' => 'johndoe@mail.com', 'token' => 'some' } }
```

Comment:
@tauil ok, but this token comes from Google. The preceding 'Bearer' is given when authenticating, so it should appear in the controller specs (the `xit` one), right?